frontend/packages/kubevirt-plugin/src/components/dashboards-page/overview-dashboard/inventory.tsx

```diff
@@ -41,7 +42,14 @@ const VM_STATUS_GROUP_MAPPER = {
   ],
 };
 
-export const getVMStatusGroups: StatusGroupMapper = (vms, { pods, migrations }) => {
+export const getVMStatusGroups: StatusGroupMapper = (
+  vms,
+  {
+    vmis,
```

Comment:
you need to add vmi resources to plugin.tsx too https://github.com/openshift/console/blob/master/frontend/packages/kubevirt-plugin/src/plugin.tsx#L200

Reply:
OK, i see its added, sorry :)